2025-10-21 13:27:12 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:27:12 - hestia - INFO - Exception handlers registrados
2025-10-21 13:27:12 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:27:12 - hestia - INFO - Exception handlers registrados
2025-10-21 13:27:22 - hestia - ERROR - Exceção não tratada: UndefinedError: 'obter_mensagens' is undefined - Path: /cadastro_contratante - IP: 127.0.0.1
Traceback (most recent call last):
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/middleware/errors.py", line 164, in __call__
    await self.app(scope, receive, _send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/fastapi/routing.py", line 123, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/fastapi/routing.py", line 109, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/fastapi/routing.py", line 389, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/fastapi/routing.py", line 288, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/routes/public_routes.py", line 271, in cadastro_contratante_form
    return templates.TemplateResponse("cadastro_contratante.html", {"request": request})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/templating.py", line 210, in TemplateResponse
    return _TemplateResponse(
           ^^^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/starlette/templating.py", line 41, in __init__
    content = template.render(context)
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "templates/cadastro_contratante.html", line 4, in top-level template code
    {% set dados = dados if dados is defined else {} %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "templates/base_simples.html", line 31, in top-level template code
    {{ obter_mensagens(request) | tojson }}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/Externo/Ifes/hestia/.venv/lib/python3.11/site-packages/jinja2/utils.py", line 92, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'obter_mensagens' is undefined
2025-10-21 13:27:52 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:27:52 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:27:52 - hestia - INFO - Exception handlers registrados
2025-10-21 13:27:52 - hestia - INFO - Exception handlers registrados
2025-10-21 13:27:58 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:27:58 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:27:58 - hestia - INFO - Exception handlers registrados
2025-10-21 13:27:58 - hestia - INFO - Exception handlers registrados
2025-10-21 13:28:26 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 13:28:26 - hestia - INFO - Exception handlers registrados
2025-10-21 14:38:57 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:38:57 - hestia - INFO - Exception handlers registrados
2025-10-21 14:39:23 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:39:23 - hestia - INFO - Exception handlers registrados
2025-10-21 14:40:08 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:40:08 - hestia - INFO - Exception handlers registrados
2025-10-21 14:40:18 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:40:18 - hestia - INFO - Exception handlers registrados
2025-10-21 14:40:34 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:40:34 - hestia - INFO - Exception handlers registrados
2025-10-21 14:40:46 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:40:46 - hestia - INFO - Exception handlers registrados
2025-10-21 14:40:56 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:40:56 - hestia - INFO - Exception handlers registrados
2025-10-21 14:41:07 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:41:07 - hestia - INFO - Exception handlers registrados
2025-10-21 14:41:33 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:41:33 - hestia - INFO - Exception handlers registrados
2025-10-21 14:41:42 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:41:42 - hestia - INFO - Exception handlers registrados
2025-10-21 14:41:49 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:41:49 - hestia - INFO - Exception handlers registrados
2025-10-21 14:41:57 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:41:57 - hestia - INFO - Exception handlers registrados
2025-10-21 14:42:05 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:42:05 - hestia - INFO - Exception handlers registrados
2025-10-21 14:42:11 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:42:11 - hestia - INFO - Exception handlers registrados
2025-10-21 14:42:54 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:42:55 - hestia - INFO - Exception handlers registrados
2025-10-21 14:43:00 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:43:00 - hestia - INFO - Exception handlers registrados
2025-10-21 14:43:08 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:43:08 - hestia - INFO - Exception handlers registrados
2025-10-21 14:43:15 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:43:16 - hestia - INFO - Exception handlers registrados
2025-10-21 14:43:23 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:43:23 - hestia - INFO - Exception handlers registrados
2025-10-21 14:43:48 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:43:48 - hestia - INFO - Exception handlers registrados
2025-10-21 14:43:56 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:43:57 - hestia - INFO - Exception handlers registrados
2025-10-21 14:44:07 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:44:07 - hestia - INFO - Exception handlers registrados
2025-10-21 14:44:17 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 14:44:17 - hestia - INFO - Exception handlers registrados
2025-10-21 17:38:52 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 17:38:52 - hestia - INFO - Exception handlers registrados
2025-10-21 23:10:59 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-21 23:11:00 - hestia - INFO - Exception handlers registrados
2025-10-21 23:24:00 - hestia - WARNING - HTTPException 404: Not Found - Path: /politica-privacidade - IP: 127.0.0.1
2025-10-21 23:24:02 - hestia - WARNING - HTTPException 404: Not Found - Path: /termos-uso - IP: 127.0.0.1
2025-10-21 23:24:04 - hestia - WARNING - HTTPException 404: Not Found - Path: /contato - IP: 127.0.0.1
2025-10-21 23:24:36 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:24:48 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:24:48 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:00 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:00 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/js/cadastro_cuidador.js - IP: 127.0.0.1
2025-10-21 23:25:00 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/css/cadastro_contratante.css - IP: 127.0.0.1
2025-10-21 23:25:23 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:23 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:25 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:33 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:33 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:25:35 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:27:46 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/js/cadastro_cuidador.js - IP: 127.0.0.1
2025-10-21 23:27:46 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/css/cadastro_contratante.css - IP: 127.0.0.1
2025-10-21 23:29:05 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:29:08 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:29:08 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/js/cadastro_cuidador.js - IP: 127.0.0.1
2025-10-21 23:29:08 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/css/cadastro_contratante.css - IP: 127.0.0.1
2025-10-21 23:29:26 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:29:26 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/js/cadastro_cuidador.js - IP: 127.0.0.1
2025-10-21 23:29:26 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/css/cadastro_contratante.css - IP: 127.0.0.1
2025-10-21 23:35:01 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:35:01 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/js/cadastro_cuidador.js - IP: 127.0.0.1
2025-10-21 23:35:01 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/css/cadastro_contratante.css - IP: 127.0.0.1
2025-10-21 23:35:01 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:35:01 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/css/cadastro_contratante.css - IP: 127.0.0.1
2025-10-21 23:35:01 - hestia - WARNING - HTTPException 404: Not Found - Path: /static/js/cadastro_cuidador.js - IP: 127.0.0.1
2025-10-21 23:37:27 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:37:28 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:17 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:21 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:27 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:29 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:31 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:35 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:47 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:48 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:54 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:55 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:41:56 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:42:00 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:42:02 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:42:41 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-21 23:42:45 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:11:52 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:11:53 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:11:57 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:11:57 - hestia - INFO - Exception handlers registrados
2025-10-22 00:12:02 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:07 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:08 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:10 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:15 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:52 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:54 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:12:57 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:13:03 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:13:05 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:13:06 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:13:09 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:15:58 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:15:59 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:16:05 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:20:18 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:33:06 - hestia - WARNING - HTTPException 404: Not Found - Path: /.well-known/appspecific/com.chrome.devtools.json - IP: 127.0.0.1
2025-10-22 00:44:24 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:44:24 - hestia - INFO - Exception handlers registrados
2025-10-22 00:44:30 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:44:30 - hestia - INFO - Exception handlers registrados
2025-10-22 00:44:54 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:44:54 - hestia - INFO - Exception handlers registrados
2025-10-22 00:47:03 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:47:05 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:47:05 - hestia - ERROR - Erro ao inserir usuário: UNIQUE constraint failed: usuario.cpf
Traceback (most recent call last):
  File "/Volumes/Externo/Ifes/hestia/data/usuario/usuario_repo.py", line 44, in inserir
    cursor.execute(INSERIR_USUARIO, parametros)
sqlite3.IntegrityError: UNIQUE constraint failed: usuario.cpf
2025-10-22 00:47:35 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:47:36 - hestia - ERROR - Erro ao inserir usuário: UNIQUE constraint failed: usuario.cpf
Traceback (most recent call last):
  File "/Volumes/Externo/Ifes/hestia/data/usuario/usuario_repo.py", line 44, in inserir
    cursor.execute(INSERIR_USUARIO, parametros)
sqlite3.IntegrityError: UNIQUE constraint failed: usuario.cpf
2025-10-22 00:48:11 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:48:11 - hestia - INFO - Exception handlers registrados
2025-10-22 00:48:12 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:48:12 - hestia - INFO - Exception handlers registrados
2025-10-22 00:48:18 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 00:48:19 - hestia - ERROR - Erro ao inserir usuário: UNIQUE constraint failed: usuario.cpf
Traceback (most recent call last):
  File "/Volumes/Externo/Ifes/hestia/data/usuario/usuario_repo.py", line 45, in inserir
    cursor.execute(INSERIR_USUARIO, parametros)
sqlite3.IntegrityError: UNIQUE constraint failed: usuario.cpf
2025-10-22 07:49:30 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
2025-10-22 07:49:31 - hestia - ERROR - Erro ao inserir usuário: UNIQUE constraint failed: usuario.cpf
Traceback (most recent call last):
  File "/Volumes/Externo/Ifes/hestia/data/usuario/usuario_repo.py", line 45, in inserir
    cursor.execute(INSERIR_USUARIO, parametros)
sqlite3.IntegrityError: UNIQUE constraint failed: usuario.cpf
2025-10-22 11:38:44 - hestia - INFO - Sistema de logging iniciado - Nível: INFO
